types/onfido-sdk-ui/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for onfido-sdk-ui 6.0
+// Project: https://github.com/onfido/onfido-sdk-ui#readme
+// Definitions by: Liam Read <https://github.com/LiamDotPro>
+//                 aktary <https://github.com/aktary>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface iOnfidoResponse {
```

Comment:
The lint rule about not starting with "I" is actually suggesting dropping the prefix, rather than making it lowercase.

Reply:
Removed the Prefix and updated the tests :+1: